test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -169,36 +168,38 @@ public void testInitOfUrlMetadataListerDocIdIsUrlFalse() throws Exception {
   public void testInitOfUrlMetadataListerDocIdIsUrlTrue() throws Exception {
     Map<String, String> moreEntries = new HashMap<String, String>();
     moreEntries.put("db.modeOfOperation", "urlAndMetadataLister");
-    moreEntries.put("db.modeOfOperation.urlAndMetadataLister.columnName", "ur");
     moreEntries.put("docId.isUrl", "true");
     final Config config = createStandardConfig(moreEntries);
     DatabaseAdaptor adaptor = new DatabaseAdaptor();
     adaptor.initConfig(config);
     adaptor.init(TestHelper.createConfigAdaptorContext(config));
   }
 
+  // TODO(bmj): call initConfig() to set defaults for most properties.
```

Comment:
Did you look into this? We're already calling initConfig here. We could easily statically import setConfigValue from TestHelper, and replace moreEntries.put with setConfigValue without much verbosity.